unittests/snapshot_tests.cpp

```diff
@@ -727,18 +722,15 @@ BOOST_AUTO_TEST_CASE_TEMPLATE(test_kv_snapshot, SNAPSHOT_SUITE, snapshot_suites)
          std::list<snapshotted_tester> sub_testers;
 
          for (int generation = 0; generation < generation_count; generation++) {
-            std::cout << "generation: " << generation << std::endl;
             // create a new snapshot child
             auto writer = SNAPSHOT_SUITE::get_writer();
             chain.control->write_snapshot(writer);
             auto snapshot = SNAPSHOT_SUITE::finalize(writer);
 
+            // Set backing_store for load snapshot
+            set_backing_store(chain, resulting_backing_store);
+
```

Comment:
after line 745, please add a #warning indicating that we need to improve the test by adding verification of the kv_object content and also need to improve the test to store to more than one key stored, so that snapshot looping is tested

Reply:
Done.